packages/storage-postgres/src/db.ts

```diff
@@ -1,40 +1,36 @@
 import { neonConfig, Pool } from '@neondatabase/serverless';
 import { drizzle } from 'drizzle-orm/neon-serverless';
 import { WebSocket } from 'ws';
-import * as schema from './schema';
+import { schema } from './schema';
 import { loadEnvConfig } from '@next/env';
 import path from 'path';
-import { drizzle as drizzleNode } from 'drizzle-orm/node-postgres';
-import { Client } from 'pg';
-import { NodePgDatabase } from 'drizzle-orm/node-postgres';
 import { NeonHttpDatabase } from 'drizzle-orm/neon-http';
+import invariant from 'tiny-invariant';
 
-type Database = NodePgDatabase<typeof schema> | NeonHttpDatabase<typeof schema>;
+type Database = NeonHttpDatabase<typeof schema>;
 
 const cwd = path.join(process.cwd(), '../../');
 const { combinedEnv } = loadEnvConfig(cwd);
 
 const connectionString =
   combinedEnv?.BRANCH_DB_URL || combinedEnv?.DEFAULT_DB_URL;
 
-if (!connectionString) {
-  throw new Error(
-    'db connectionString (BRANCH_DB_URL or DEFAULT_DB_URL) is not set',
-  );
-}
-
-let db: Database;
+invariant(
+  connectionString,
+  'db connectionString (BRANCH_DB_URL or DEFAULT_DB_URL) is not set',
+);
 
-if (process.env.NODE_ENV === 'production') {
+if (connectionString.includes('localhost')) {
+  neonConfig.wsProxy = (host) => `${host}:5433/v1`;
+  neonConfig.useSecureWebSocket = false;
+  neonConfig.pipelineTLS = false;
+  neonConfig.pipelineConnect = false;
+} else {
   neonConfig.webSocketConstructor = WebSocket;
   neonConfig.poolQueryViaFetch = true;
-  const pool = new Pool({ connectionString });
-  db = drizzle(pool, { schema });
-} else {
-  const client = new Client({ connectionString });
-  client.connect();
-  db = drizzleNode(client, { schema });
 }
```

Comment:
We need to setup websocket proxy when connection string includes localhost instead of `NODE_ENV` `production`